portfolio/src/main/webapp/index.html

```diff
@@ -1,16 +1,42 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div id="fullpage">
+		<div class="section">
+		</div>
+		<div class="section">
+		</div>
+        <div class="section">
+		</div>
+        <div class="section" id="contact">
+            <h1>Check me out!</h1>
+            <div class="socials-grid">
+                <a href="">
+                    <img src="https://via.placeholder.com/150/">
```

Comment:
Tip for later, not for this PR: [provide alternative fallback for multimedia content](https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback)